packages/react/src/components/footer/footerNavigationGroup/FooterNavigationGroup.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+// import core base styles
+import 'hds-core';
+import { useMediaQueryLessThan } from '../../../hooks/useMediaQuery';
+import styles from './FooterNavigationGroup.module.scss';
+import classNames from '../../../utils/classNames';
+
+type FooterNavigationGroupProps = React.PropsWithChildren<{
+  /**
+   * Description of the navigation group
+   * for screen readers.
+   */
+  ariaLabel?: string;
+  /**
+   * Additional class names to apply.
+   */
+  className?: string;
+  /**
+   * ID of the navigation group element.
+   */
+  id?: string;
+  /**
+   * FooterGroupHeading component to display at the top of the group. On smaller screens only this will be displayed.
+   * @example
+   * headingLink={<Footer.GroupHeading
+            href="https://yourpath.com"
+            label="Main Page"
+            variant={FooterVariant.Navigation}
+          />}
+   */
+  headingLink: React.ReactNode;
```

Comment:
FooterUtilityGroup component has this Footer.GroupHeading component as well but it is placed inside the component as part of children. Is it better to have it as a prop like in here for consistency's sake?